feat: improve the website with AI 🤖 (#495)
countr/website: content/docs/01-features/09-notifications.md

# Notifications

> If you want to stay up-to-date in your counting channel, or someone else's counting channel, you can set up Countr to send you a DM whenever you'd like.

Reviewer: ```suggestion
Countr can send you a DM to help you stay up-to-date in counting channels everywhere!
```
